tests/integration/defs/examples/visual_gen/test_visual_gen_multi_gpu.py

```
@@ -175,6 +192,265 @@ def _skip_if_insufficient_gpus_for_parallel(parallel):
+        f"--ntasks={WAN22_LPIPS_MULTINODE_WORLD_SIZE}",
+        f"--ntasks-per-node={WAN22_LPIPS_MULTINODE_GPUS_PER_NODE}",
+        "--export=ALL",
+        sys.executable,
+        "-m",
+        "pytest",
+        "-q",
+        "-s",
+        nodeid,
+    ]
+    try:
+        result = subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)